src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,820 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Created with Cambalache 0.10.2 -->
+<interface>
+  <!-- interface-name dialog-vkbasalt.ui -->
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="VkBasaltDialog" parent="AdwWindow">
+    <property name="default-width">500</property>
+    <property name="default-height">600</property>
+    <property name="modal">True</property>
+    <property name="title" translatable="yes" comments="vkBasalt is a Vulkan post processing layer for Linux">vkBasalt Settings</property>
+    <child>
+      <object class="GtkShortcutController">
+        <child>
+          <object class="GtkShortcut">
+            <property name="trigger">Escape</property>
+            <property name="action">action(window.close)</property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="AdwHeaderBar">
+            <property name="show-end-title-buttons">False</property>
```

Comment:
```suggestion
            <property name="show-start-title-buttons">False</property>
            <property name="show-end-title-buttons">False</property>
```

Also, why do we even use AdwHeaderBar where it's very much unneccessary, like here? But I digress.